_includes/layouts/kids.njk

```diff
@@ -5,71 +5,123 @@ templateClass: tmpl-kids
+                const computerMove = data.recommendation
+                move(board, computerMove, 'X')
+                const cell = document.getElementById(computerMove.toString())
+                cell.innerText = 'X'
+                setPlayerTurn('O')
+            })
+        }
+        const clickCell = (cellId) => {
+            console.info("inside setCell", cellId)
+            if (gameState.playing) { // human is always O
+                const cell = document.getElementById(cellId)
+                if (board.charAt([parseInt(cellId)]) === '-') {
+                    cell.innerText = 'O'
+                    setPlayerTurn('X')
+                    computerTurn()
+                }
+            }
+        }
+        const setBoard = (newBoard) => {
+            board = newBoard
+        }
+        const clearCells = () => {
+            const cellIds = [
+                '1',
+                '2',
+                '3',
+                '4',
+                '5',
+                '6',
+                '7',
+                '8',
+                '9'
+            ]
+            cellIds.forEach(cellId => {
+                const cell = document.getElementById(cellId)
+                cell.innerText = ''
+            })
+        }
+        const clearBoard = () => {
+            board = '---------'
+        }
+        const move = (board, player, location) => {
+            return board.replaceAt(location, player)
+        }
+        const checkGameStatus = () => {
+            // top row win
+            if ((board.charAt(0) !== '-') && (board.charAt(1) === board.charAt(2)) && (board.charAt(2) === board.charAt(3)) {
+                gameState.winner = gameState.playerTurn
+                gameState.playing = false
             }
-            const tableElementNumbers = Array.from({
-                length: 10
-            }, (_, i) => numberToWord[i + 1])
-            tableElementNumbers.forEach((cell) => document.getElementById(cell).innerText = 'X')
+            // middle row win
+                if ((board.charAt(3) !== '-') && (board.charAt(3) === board.charAt(4)) && (board.charAt(4) === board.charAt(5)) {}
+            // bottom row win
+                if ((board.charAt(6) !== '-') && (board.charAt(6) === board.charAt(7)) && (board.charAt(7) === board.charAt(8)) {}
+            // left col win
+                if ((board.charAt(0) !== '-') && (board.charAt(0) === board.charAt(3)) && (board.charAt(3) === board.charAt(6)) {}
+            // middle col win
+                if ((board.charAt(1) !== '-') && (board.charAt(1) === board.charAt(4)) && (board.charAt(4) === board.charAt(7)) {}
+            // right col win
+                if ((board.charAt(2) !== '-') && (board.charAt(2) === board.charAt(5)) && (board.charAt(5) === board.charAt(8)) {}
         }
-        function play() {
-            // play Tic Tac Toe
-            console.log("board.one", board.one)
-            console.log("play TTT")
+        const play =() => {
```

Comment:
@twalker I'm interested in a particular kind of question I don't have the answer to. I was curious what you think might make sense. So, imagine a team of bright people, but not that experienced with tech stuff. In fact, maybe one of them won a nobel prize in poetry, and can jam with the best of em! But we still have to make decisions about workflow, and about code repo patterns. Here is why I'm interested in this. With software engineers, they tend to be more comfortable with switching processes, switching from one technology to another, switching from one way of doing things to another. But, that trait of switching stuff, is really a software engineering thing, it's not a bright person thing. And this means we have to be very careful and intentional about how we design our processes, workflows, and code patterns as a technical writing team.

Reply:
So, with your experience using Eleventy so far, do you have some thoughts to share about what to pay attention to when starting a new project?

Reply:
In hindsight, I chose too complex of a starter project in Eleventy. I don't regret choosing Eleventy over Jekyll, but should have started with more of a bare bones Eleventy starter with minimal configuration. I thought having a more fully functional starter project would be the most flexible, but suspect it made the project less approachable to contribute to. 

If I had a chance to restart, I'd started us with a more basic setup, knowing the more advanced functionality, such as tags, multiple layouts, etc., would be available if we needed it. The team could grow into more advanced features organically, without all the bells and whistles already in place.

I should have considered my primary audience more than satisfying my own curiosity. So, the lesson to me is to keep the readers at the center of decisions. The same spirit echos in the "Keep It Stupid Simple" (KISS) principle, as well as the "You Ain't Gonna Need It" (YAGNI) principle.